velox/common/memory/Memory.cpp

```diff
@@ -58,12 +61,6 @@ MemoryManager::MemoryManager(const MemoryManagerOptions& options)
               .checkUsageLeak = options.checkUsageLeak,
               .debugEnabled = options.debugEnabled})} {
   VELOX_CHECK_NOT_NULL(allocator_);
-  if (arbitrator_ != nullptr) {
-    VELOX_CHECK_EQ(
```

Comment:
```
s/VELOX_CHECK_EQ/VELOX_CHECK_LE/ ?
```

Reply:
I removed it because it is already enforced by std::min(..., ...) in the constructor init list.